recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -40,7 +40,7 @@ INI_UNCOMMENT_ASSIGNMENTS_append_mx8mq = " \
 "
 
 uncomment() {
-    if ! (grep "^#$1" $2); then
+    if ! grep -q "^#$1" $2 && ! grep -q "^$1" $2; then
```

Comment:
I think if the line is already uncommented there should be a `bbdebug` and return.

Reply:
We didn't do that as some (e.g., i.MX6QDL) will run this twice. So the code guarantee it is as intended (uncommented).